scipy/special/_faddeeva.cxx

```diff
@@ -63,4 +64,55 @@ npy_cdouble faddeeva_dawsn_complex(npy_cdouble zp)
     return npy_cpack(real(w), imag(w));
 }
 
+/*
+ * A wrapper for a normal CDF for complex argument
+ */
+
+npy_cdouble faddeeva_ndtr(npy_cdouble zp)
+{
+    complex<double> z(zp.real, zp.imag);
+    z *= NPY_SQRT1_2;
+    complex<double> w = 0.5 * Faddeeva::erfc(-z);
+    return npy_cpack(real(w), imag(w));
+}
+
+/*
+ * Logarithm of the normal CDF, with improved accuracy for z->-inf.
+ * This is essentially a copy of log_ndtr routine from cephes/ndtr.c
+ * 
+ * For Re(a) > -20, use the existing ndtr technique and take a log.
+ * For Re(a) <= -20, use the asymptotic series of A&S 26.2.12
+ * (equivalently, A&S 7.1.23)
+ */
+
+npy_cdouble faddeeva_log_ndtr(npy_cdouble zp)
+{
+
+    complex<double> a(zp.real, zp.imag), result;
+    complex<double> log_LHS,        /* we compute the left hand side of the approx (LHS) in one shot */
+        last_total = 0,        /* variable used to check for convergence */
+        right_hand_side = 1,    /* includes first term from the RHS summation */
+        numerator = 1,        /* numerator for RHS summand */
+        denom_factor = 1,    /* use reciprocal for denominator to avoid division */
+        denom_cons = 1.0 / (a * a);    /* the precomputed division we use to adjust the denominator */
+        double sign = 1;
+        long i = 0;
+
+    if (a.real() > -20) {
```

Comment:
Or `std::abs(a) > 20`, the series seems convergent?
For instance, `erfc(-1 + 30j) =~ (-1.38e+388, 4.88e+388)`, too big for floating point but representable in log space.